### PR TITLE
Fix keepalive fail when NoScript/AD-blocking is used

### PR DESCRIPTION
**Pull request by Fedik**
Pull Request for Issue #40157 .

### Summary of Changes
Change `sendBeacon()` to `fetch()`

### Testing Instructions
Please follow #40157

### Actual result BEFORE applying this Pull Request
You loged out

### Expected result AFTER applying this Pull Request
All works

### Link to documentations
Please select:
- [x] No documentation changes for docs.joomla.org needed
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: build/media_source/system/js/keepalive.es6.js [hunk 1]
```diff
@@ -26,4 +26,4 @@ if (keepAliveUri === '') {
   keepAliveUri = `${(systemPaths ? `${systemPaths.root}/index.php` : window.location.pathname)}?option=com_ajax&format=json`;
 }
 
-setInterval(() => navigator.sendBeacon(keepAliveUri), keepAliveInterval);
+setInterval(() => fetch(keepAliveUri, {method: 'POST'}), keepAliveInterval);
```
Comment (richard67) [inline build/media_source/system/js/keepalive.es6.js]:
```suggestion
setInterval(() => fetch(keepAliveUri, { method: 'POST' }), keepAliveInterval);
```
This should fix javascript cs in drone.

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 02e942a18f42561f25db19737608c13085cb15f4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40195">issues.joomla.org/tracker/joomla-cms/40195</a>.</sub>

**Comment by laoneo** [comment]
Is this not a duplicate of https://github.com/joomla/joomla-cms/pull/40221?

**Comment by laoneo** [comment]
I was sure that I miss here something. Thanks..

**Comment by laoneo** [comment]
I have tested this item :white_check_mark: successfully on 02e942a18f42561f25db19737608c13085cb15f4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40195">issues.joomla.org/tracker/joomla-cms/40195</a>.</sub>

**Comment by obuisard** [comment]
Thank you Fedir @Fedik !

**Pull request merged**